.github/workflows/ci-unit-tests.yml

```diff
@@ -28,3 +28,15 @@ jobs:
         file: cover.out
         fail_ci_if_error: true
         verbose: true
+
+  golangci-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.33
+          only-new-issues: true
```

Comment:
Could you please explain what `only new issues` means? Does it mean it only scans changed files (if so how does it determine that, against master?), or that it reports errors previously unseen (if so where does it store the state?) ?

Reply:
This has been dropped since all linters are currently passing. 
Per  https://golangci-lint.run/usage/configuration/ 
```
 # Show only new issues: if there are unstaged changes or untracked files,
  # only those changes are analyzed, else only changes in HEAD~ are analyzed.
  # It's a super-useful option for integration of golangci-lint into existing
  # large codebase. It's not practical to fix all existing issues at the moment
  # of integration: much better don't allow issues in new code.
  # Default is false.
```
This is meant to allow easier integration into a large codebase by only reporting errors if the added code contains them. It does not report previously seen linter errors (those that are already in master). 
This has been dropped though because all outstanding linter issues have been fixed.